File: src/targets/java/unirest/fixtures/nested.java
<<<<<<< HEAD
HttpResponse<String> response = Unirest.get("https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value")
=======
HttpResponse<String> response = Unirest.get("http://mockbin.com/har?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value")
>>>>>>> upstream/master
  .asString();

File: src/targets/node/unirest/fixtures/nested.js
const unirest = require('unirest');

<<<<<<< HEAD
const req = unirest('GET', 'https://httpbin.org/anything');
=======
const req = unirest('GET', 'http://mockbin.com/har');
>>>>>>> upstream/master

req.query({
  'foo[bar]': 'baz,zap',
  fiz: 'buz',
  key: 'value'
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});